scripts/make-release.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -e
+
+if [[ "$1" = "--status" ]]; then
+    gh workflow view release --repo unisonweb/unison
+    gh workflow view release --repo unisonweb/homebrew-unison
+fi
+
+usage() {
+    echo "NOTE: must be run from the root of the project."
+    echo "Usage: $0 VERSION [TARGET]"
+    echo "VERSION: The version you're releasing, e.g. M4a"
+    echo "TARGET: The revision to make the release from, defaults to 'trunk'"
+    echo ""
+    echo "E.g."
+    echo "$0 M4a"
+}
+
+if [[ -z "$1"  ]] ; then
+  usage
+  exit 1
+fi
+
+if ! command -V "gh" >/dev/null 2>&1; then
+   echo "Required command \`gh\` not found, find installation instructions here: https://cli.github.com/manual/installation"
+   exit 1
+fi
+
+if ! [[ "$1" =~ ^M[0-9]+[a-z]?$ ]] ; then
+ echo "Version tag must be of the form 'M4' or 'M4a'"
+ usage
+ exit 1
+fi
+
+version="${1}"
+prev_version=$(./scripts/previous-tag.sh "$version")
+target=${2:-trunk}
+
+prereleaseFlag=""
+
+if [[ "$1" =~ ^M[0-9]+[a-z]$ ]] ; then
+    prereleaseFlag="--prerelease"
+fi
+
+echo "Creating release in unison-local-ui..."
+gh release create "release/${version}" --repo unisonweb/unison-local-ui --target main --generate-notes $prereleaseFlag --notes-start-tag "release/${prev_version}"
+
+echo "Kicking off release workflow in unisonweb/unison"
+gh workflow run release --repo unisonweb/unison --field "version=${version}" --field "target=${target}" --ref cp/automate-minor-releases
```

Comment:
Reminder to Chris to remove all of these --refs before merging.

```suggestion
gh workflow run release --repo unisonweb/unison --field "version=${version}" --field "target=${target}"
```